lms/templates/main.html

```diff
@@ -39,12 +39,6 @@
       </script>
   % endif
 
-  <%
-    jsi18n_path = "js/i18n/{language}/djangojs.js".format(language=LANGUAGE_CODE)
```

Comment:
Will moving this out of the head cause it to flash and change the language of the site after initial paint?